CHANGELOG.md

```diff
@@ -4,7 +4,10 @@
 
 [Full Changelog](https://github.com/voxpupuli/json-schema/compare/v2.8.1...v3.0.0)
 
-**Breaking changes:**
+### Fixed
```

Comment:
@chelnak please undo this. We will automatically generate the changelog during the next release